src/tools/ceph_dencoder.cc

```diff
@@ -33,7 +33,7 @@
 #define TYPE_FEATUREFUL_NOCOPY(t)
 #define TYPE_NOCOPY(t)
 #define MESSAGE(t)
-#include "types.h"
+#include "include/types.h"
```

Comment:
This is wrong -- it should be ```test/encoding/types.h``` -- right now ceph-dencoder is broken since it isn't including any version controlled types.